src/style.css

```diff
@@ -0,0 +1,31 @@
+@import "./styles/card.css";
+@import "./styles/stars.css";
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.button {
+  font-size: 14px;
+  font-weight: 700;
+  text-transform: uppercase;
+  padding: 11px 0;
+  color: #fff;
+  background-color: #00acdc;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  cursor: pointer;
+  transition: all 0.2s linear;
+}
+
+.button:hover {
+  color: #00acdc;
+  background-color: #fff;
+}
+
+:root {
+  --color-text-dark: #060b35;
+  --color-text-light: #616070;
+}
```

Comment:
push variables in separate file, import it in main style sheet before content styles